modules/rostests/apitests/user32/user32_apitest.rc

```diff
@@ -11,6 +11,7 @@ TESTCURSOR CURSOR "test.cur"
 IDR_ICONS_PNG RCDATA "ROS.ico"
 IDR_ICONS_NORMAL RCDATA "sysicon.ico"
 IDR_EXE_NORMAL RCDATA "cpimg2e.exe"
+IDR_DLL_NORMAL RCDATA "image.dll"
```

Comment:
If the file is compiled from source, avoid adding a binary of it in the source tree. Use CMake dependencies.

Reply:
If we use CMake dependencies, then we will not be certain that the resource file is compiled with MSVC's rc.exe Resource Compiler, so that is the reason for including a DLL that has a known source compiled type of MSVC. If you know of a way to guarantee the use of the MSVC compiler even under GCC, then please let me know. Thanks.

Reply:
Theoretically we could add the test conditionally in the CMakeLists.txt with some `ifdef MSVC...` only and not for GCC. But that would weaken the test a bit of course. Therefore I do think atm, that Dougs current solution is appropriate here.